src/odemis/util/__init__.py

```diff
@@ -147,27 +147,37 @@ def rect_intersect(ra, rb):
 
 
 def perpendicular_distance(start, end, point):
-    """ Return the perpendicular distance between the line described by start and end, and point """
-
+    """
+    Computes the perpendicular distance between a segment and a point (in 2D space).
```

Comment:
what is a segment?

Reply:
Actually, it's just coming from my broken english. Apparently, the official name is "line segment" (as defined in Wikipedia). => Fixed.